### PR TITLE
Prepare for 1.0.0-beta2 release

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
Let's get some of the super useful changes we've been making into the hands of our users!

# Review

- [x] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

File: scripts/publish.py [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "wasmer-c-api": set(["wasmer", "wasmer-compiler", "wasmer-compiler-cranelift", "wasmer-compiler-singlepass",
                          "wasmer-compiler-llvm", "wasmer-emscripten", "wasmer-engine", "wasmer-engine-jit",
                          "wasmer-engine-native", "wasmer-engine-object-file", "wasmer-wasi", "wasmer-types"]),
+    "wasmer-middlewares": set(["wasmer", "wasmer-types", "wasmer-vm"]),
```
Comment (nlewycky) [inline scripts/publish.py:52]: (No action required:) huh, so beta1 didn't have the middleware? Were the users using the revision from git then?

**Review by nlewycky — APPROVED** [review]

File: scripts/publish.py (hunk 1 above)
Comment (jubianchi) [reply scripts/publish.py:52]: I think yes.

**Review by jubianchi — APPROVED** [review]

File: scripts/publish.py (hunk 1 above)
Comment (MarkMcCaskey) [reply scripts/publish.py:52]: Yeah, it seems so!  We updated the version but did not publish it last time

**Pull request merged**